lms/djangoapps/mobile_api/notifications/views.py

```diff
@@ -0,0 +1,50 @@
+from django.conf import settings
+from rest_framework import status
+from rest_framework.response import Response
+
+from edx_ace.push_notifications.views import GCMDeviceViewSet as GCMDeviceViewSetBase
```

Comment:
This class is imported from django-push-notifications into edx-ace so that there is no direct dependency, and in case of need, it can be changed in edx-ace without changing the implementation in the platform.

https://github.com/openedx/edx-ace/blob/master/edx_ace/push_notifications/views/__init__.py